apps/arena/priv/config.json

```diff
@@ -315,6 +315,27 @@
       ],
       "effects_to_apply": []
     },
+    {
+      "name": "warp",
+      "cooldown_mechanism": "time",
+      "cooldown_ms": 2000,
+      "execution_duration_ms": 800,
+      "inmune_while_executing": true,
+      "activation_delay_ms": 600,
+      "is_passive": false,
+      "autoaim": false,
+      "can_pick_destination": true,
+      "stamina_cost": 1,
+      "mechanics": [
+        {
+          "teleport": {
+            "range": 1000,
+            "duration_ms": 300
```

Comment:
I think we should include the radius of the ability so we can displayed it in the client. Or maybe we can use the hitbox radius in the client to display it.

 What do you think?

Reply:
ooooh right! yeah we should use the body size for the inner circle indicator